sdk/avalon_sdk/connector/blockchains/ethereum/ethereum_wrapper.py

```diff
@@ -190,7 +192,7 @@ def sign_execute_raw_transaction(self, tx_dict):
             tx_dict, private_key=self.__eth_private_key)
         tx_hash = self.__w3.eth.sendRawTransaction(signed_tx.rawTransaction)
         tx_receipt = self.__w3.eth.waitForTransactionReceipt(
-            tx_hash.hex(), 120)
+            tx_hash.hex(), timeout=10, poll_latency=0.2)
```

Comment:
Add comment about parameter poll_latency.

Reply:
Yes will add. Added in the commit log. Forgot here.